app/models/save_and_return_settings_updater.rb

```diff
@@ -0,0 +1,56 @@
+class SaveAndReturnSettingsUpdater
+  attr_reader :save_and_return_settings, :service_id, :service_name
+
+  def initialize(save_and_return_settings:, service_id:, service_name:)
+    @save_and_return_settings = save_and_return_settings
+    @service_id = service_id
+    @service_name = service_name
+  end
+
+  def create_or_update!
+    ActiveRecord::Base.transaction do
+      save_config_save_and_return
+    end
+  end
+
+  private
+
+  def save_config_save_and_return
+    if save_and_return_settings.save_and_return_enabled?
+      create_or_update_service_configuration(config: 'SAVE_AND_RETURN', deployment_environment: 'dev', value: 'enabled')
+      create_or_update_service_configuration(config: 'SAVE_AND_RETURN', deployment_environment: 'production', value: 'enabled')
```

Comment:
I had to hardcode the value `enabled` in here as this is what the MDP is expecting currently.

We should check for presence of the `SAVE_AND_RETURN` env var in the MDP to be consistent with other checks in the MDP but this will do for now to get it working e2e without having to bump the MDP.